### PR TITLE
[Feature] Added add form for all the models

### PR DESCRIPTION
**Pull request by RahulRimal**

### CODE REVIEW

File: src/components/Drawer.jsx [hunk 1]
```diff
@@ -70,6 +70,10 @@ const Drawer = ({ drawerWidth }) => {
               prefix={<MdOutlineCreditCardOff />}
               onClick={() => navigate("/company")}
             />
+            <ListItem
+              prefix={<MdOutlineCreditCardOff />}
+              onClick={() => navigate("/add-data")}
```
Comment (Jeevan1) [inline src/components/Drawer.jsx:75]: change path name

File: src/App.jsx [hunk 2]
```diff
@@ -118,6 +130,10 @@ const App = () => {
           path: "/profile",
           element: <EmployeeProfile />,
         },
+        {
+          path: "/add-data",
```
Comment (Jeevan1) [inline src/App.jsx:134]: remove this path

File: src/components/Drawer.jsx [hunk 3]
```diff
@@ -115,6 +119,11 @@ const Drawer = ({ drawerWidth }) => {
               text="Report"
               onClick={() => navigate("/report")}
             />
+            <ListItem
+              prefix={<MdOutlineCreditCardOff />}
+              text="Forms"
+              onClick={() => navigate("/add-data")}
```
Comment (Jeevan1) [inline src/components/Drawer.jsx:125]: change path name

**Review by Jeevan1 — CHANGES_REQUESTED** [review]

File: src/components/Drawer.jsx (hunk 1 above)
Comment (RahulRimal) [reply src/components/Drawer.jsx:75]: done

File: src/App.jsx (hunk 2 above)
Comment (RahulRimal) [reply src/App.jsx:134]: done

File: src/components/Drawer.jsx (hunk 3 above)
Comment (RahulRimal) [reply src/components/Drawer.jsx:125]: done

**Pull request merged**